### PR TITLE
Update Simplified Chinese localization

### PR DESCRIPTION
**Pull request by Jerry23011**
### Changes
- Add Simplified Chinese localization for profile accessibility.

@nixzhu 这回的还真不少 😄

### CODE REVIEW

File: IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings [hunk 1]
```diff
@@ -521,44 +521,44 @@
...
+"accessibility.tabs.profile.header-image.label" = "横幅图片";
+"accessibility.tabs.profile.header-image.hint" = "显示更大尺寸。";
+"accessibility.tabs.profile.post-count.hint" = "滚动到列表。";
+"accessibility.tabs.profile.following-count.hint" = "前往列表。";
+"accessibility.tabs.profile.follower-count.hint" = "前往列表。";
+"accessibility.tabs.profile.picker.statuses" = "嘟文";
+"accessibility.tabs.profile.picker.favorites" = "喜欢";
+"accessibility.tabs.profile.picker.bookmarks" = "书签";
+"accessibility.tabs.profile.picker.followed-tags" = "标签";
+"accessibility.tabs.profile.picker.posts-and-replies" = "读文和回复";
```
Comment (nixzhu) [inline IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings:550]: 嘟文和回复

File: IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings [hunk 2]
```diff
@@ -521,44 +521,44 @@
...
+"accessibility.tabs.profile.picker.lists" = "列表";
+"accessibility.tabs.profile.user-notifications.label" = "接收通知";
+"accessibility.tabs.profile.user-reblogs.label" = "显示转发";
+"accessibility.tabs.profile.fields.verified.label" = "已认证";
+"accessibility.tabs.profile.fields.container.label" = "用户自定义字段";
+"accessibility.status.spoiler-full-content" = "完整内容";
+"accessibility.status.a-boosted-b-%@-%@" = "%@ 转发了 %@";
+"accessibility.status.a-replied-to-%@" = "%@ 回复给";
+"accessibility.image.alt-text-%@" = "图片描述文本：%@";
+"accessibility.image.alt-text-more.label" = "更多替代文本可用";
```
Comment (nixzhu) [inline IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings:561]: 更多描述文本可用

File: IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings (hunk 1 above)
Comment (Jerry23011) [reply IceCubesApp/Resources/Localization/zh-Hans.lproj/Localizable.strings:550]: 不好意思，typo

**Comment by nixzhu** [comment]
LGTM

**Pull request merged**